Cargo.toml

```
@@ -38,7 +38,9 @@ diesel_migrations = "2.1.0"
 default = []
 mysql = ["diesel/mysql_backend", "mysql_async", "mysql_common", "futures-channel", "tokio"]
 postgres = ["diesel/postgres_backend", "tokio-postgres", "tokio", "tokio/rt"]
-async-connection-wrapper = []
+sqlite = ["diesel/sqlite", "sync-connection-wrapper"]
+sync-connection-wrapper = ["tokio/rt"]
+async-connection-wrapper = ["tokio/net"]
```

```suggestion
async-connection-wrapper = ["tokio/rt"]
```

should be enough for that feature, right?